src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowWalker/Helpers/SettingsManager.cs

```diff
@@ -0,0 +1,141 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using System.Text.Json.Serialization;
+using Microsoft.CmdPal.Ext.WindowWalker.Properties;
+using Microsoft.CmdPal.Extensions.Helpers;
+
+#nullable enable
```

Comment:
oh look another #nullable that should just live in the csproj (probably)

Reply:
fixed.